integrations/tensorflow/e2e/keras/BUILD

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//bindings/python:build_defs.oss.bzl",
+    "INTREE_TENSORFLOW_PY_DEPS",
+    "NUMPY_DEPS",
+    "iree_py_test",
+    "iree_py_test_suite",
+    "iree_vision_test_suite",
+)
+
+E2E_INTEGRATIONS_DEPS = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+    "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+]
```

Comment:
Dep variables are actually generally discouraged and should only be used where necessary. They confuse tooling. The reason we have them here is because these deps have to get swapped in OSS and Google internal (although there are actually other ways to do that that we could investigate).

Reply:
Got it. Made the fix.